### PR TITLE
customize media player

### PR DESCRIPTION
**Pull request by AJStubbsy**
## Proposed change

I recently found out that you could specify the source_list of a media player. This helped me integrate my media player with Google Assistant and solved another members problem of being able to reduce the number of sources shown to them when using the Sony Bravia integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by fabaff — APPROVED** [review]
Thanks

File: source/_docs/configuration/customizing-devices.markdown [hunk 1]
```diff
@@ -100,6 +100,9 @@ homeassistant:
       icon: mdi:kettle
     switch.rfxtrx_switch:
       assumed_state: false
+    media_player.my_media_player
```
Comment (frenck) [inline source/_docs/configuration/customizing-devices.markdown:103]: This change is actually incorrect and broken as the YAML is now invalid.

Comment (AJStubbsy) [reply source/_docs/configuration/customizing-devices.markdown:103]: I haven't included the : after media_player. I've added this as a suggestion below, I hope that's the correct way to resolve this?
```suggestion
    media_player.my_media_player:
```

Comment (frenck) [reply source/_docs/configuration/customizing-devices.markdown:103]: No, it needs a new PR to be able to resolve, as this one is merged.

**Pull request merged**